build/squire-raw.js

```diff
@@ -889,6 +889,28 @@ var deleteContentsOfRange = function ( range, root ) {
     return frag;
 };
 
+var isNodeEmpty = function ( node ) {
+    if (!node) {
+        return false;
+    }
+
+    // check for image nodes
+    if ( node.nodeType === Node.ELEMENT_NODE && node.nodeName === 'IMG' ) {
```

Comment:
Ideally the next if statement with the query selector would be able to handle the case in which the top level node is an image as well.  Does anyone know of a way to combine these two ifs?  Maybe a way to make querySelector check itself as well?